### DISCUSSION TITLE
Is this library dead?

### DISCUSSION THREAD

**Discussion by RhavoX** [rytilahti/python-miio#1897] [General]
There are more and more devices that are left unsupported and the last release was in 2022 Feb, is this library even supported?

**Comment by rytilahti** [comment]
Well, it really depends on what you mean as supported and what exactly is not working for you. The current master branch should support all modern devices out-of-the-box, but there hasn't been any releases in the past indeed and it's a known issue, see https://github.com/rytilahti/python-miio/issues/1808

**Comment by Som83** [comment]
Unfortunately not all modern devices work. I'm still waiting for supporting Xiaomi Smart Air Purifier 4 (zhimi.airp.mb5a) #1523

**Reply by rytilahti** [reply]
Did you already try controlling it [using the `genericmiot` integration](https://github.com/rytilahti/python-miio/?tab=readme-ov-file#controlling-modern-miot-devices)?
